plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml

```diff
@@ -991,6 +991,8 @@ items:
     resources:
     - secrets
     verbs:
+    - create
+    - delete
```

Comment:
I would not expect RBAC changes due to test enhancements.

Reply:
secrets `["get", "list", "create"]` is required by azure file driver since it needs to create k8s secret to store account key and get secret when mount, and secrets `delete` is not required.

Reply:
Thanks! Fixed